src/util/options.schema.json

```diff
@@ -1407,6 +1400,20 @@
             "Path to C preprocessor (cpp) to use. If empty, then automatically searched.",
           "type": "string",
           "default": ""
+        },
+        "arrays-domain": {
+          "title": "exp.arrays-domain",
+          "description":
+            "The domain that should be used for arrays. trivial/partitioned/unroll. When employing the partition array domain, make sure to enable the expRelation analysis as well. When employing the unrolling array domain, make sure to set the exp.array-unrolling-factor >0.",
+          "type": "string",
+          "default": "trivial"
+        },
+        "array-unrolling-factor": {
+          "title": "exp.array-unrolling-factor",
+          "description":
+            "Indicates how many values will the unrolled part of the unrolled array domain contain.",
+          "type": "integer",
+          "default": 0
```

Comment:
These should be moved back under `ana.base`, for example:
* `ana.base.arrays.domain` (to match `ana.base.structs.domain`),
* `ana.base.arrays.unrolling-factor`.

Reply:
Fixed on commit ec075afd3fe3b71d6d58ea278861e2a5063b2d5c (array-domain and array-unrolling factor flags moved under ana.base. All references fixed)